packages/core/test/fixtures/transaction.ts

```diff
@@ -468,6 +487,9 @@ export const validTransactions = [
                     script: '76a914851a33a5ef0d4279bd5854949174e2c65b1d450088ac',
                 },
             ],
+            inPubkeys: [
+                '3045022100d269531f120f377ed2f94f42bef893ff2fe6544ac97fb477fa291bc6cfb7647e02200983f6a5bbd4ce6cf97f571995634805a7324cc5d8353ed954fa62477b0fcd0901',
```

Comment:
i don't think this is a public key. How are the tests passing?

Reply:
This was taken from a multisig address and this test has been skipped, removed the pubkey test data as its misleading

Reply:
I have added a test as well to make sure `isPubKey` returns false for these types of addresses